test/fitnesse/testrunner/MultipleTestsRunnerTest.java

```diff
@@ -88,7 +88,7 @@ public boolean matches(Descriptor descriptor) {
     }
   }
 
-  static interface ClosableTestSystemListener extends TestSystemListener, Closeable {
+  public static interface ClosableTestSystemListener extends TestSystemListener, Closeable {
```

Comment:
The `static` keyword is redundant here. Does removing that impact the error you get in any way?

I don't mind changing this to `public`, but at least let's remove the extra static as well

Reply:
The `static` does not make a difference, it fails either way. I've removed it and replaced it with `public`.